STMF4 ETH_DMA discrepancy between various members

Many (all?) members of the STMF4 family share the same Ethernet peripherals but only the STM32F407 includes the `peripherals/eth/eth_dma.yaml` fixup leading to discrepancies in the names of some fields:

```
error[E0599]: no method named `pm` found for type `&mut stm32f4::stm32f429::ethernet_dma::dmabmr::W` in the current scope
   --> src/ethernet/mod.rs:149:18
    |
149 |                 .pm().bits(0b01)
    |                  ^^
    |
    = help: did you mean `fpm`?
```

The fix is probably to include the .yaml in all models of the F4 family? At least those sharing the same RM0090.

--- a/devices/common_patches/f4_ethernet.yaml
+++ b/devices/common_patches/f4_ethernet.yaml
@@ -1,0 +1,34 @@
+Ethernet_MAC:
+  MACFFR:
+    _modify:
+      # This field should be 2 bits wide but is only 1 bit in the SVD
+      PCF:
+        bitWidth: 2
+  # All these fields are named incorrectly in the SVD compared to RM0090
+  MACA1LR:
+    _modify:
+      MACA1LR:
+        name: MACA1L
+  MACA2HR:
+    _modify:
+      MAC2AH:
+        name: MACA2H
+  MACA2LR:
+    _modify:
+      MACA2L:
+        bitWidth: 32
+  MACA3LR:
+    _modify:
+      MBCA3L:
+        name: MACA3L
+  MACMIIDR:
+    _modify:
+      TD:
+        name: MD
+
+Ethernet_DMA:
+  DMABMR:
+    _modify:
+      # This field is named incorrectly in the SVD compared to the RM
+      RTPR:
+        name: PM

--- a/devices/stm32f407.yaml
+++ b/devices/stm32f407.yaml
@@ -23,37 +23,6 @@ _modify:
         bitWidth: 1
         access: read-write
 
-Ethernet_MAC:
-  MACFFR:
-    _modify:
-      # This field should be 2 bits wide but is only 1 bit in the SVD
-      PCF:
-        bitWidth: 2
-  # All these fields are named incorrectly in the SVD compared to RM0090
-  MACA1LR:
-    _modify:
-      MACA1LR:
-        name: MACA1L
-  MACA2HR:
-    _modify:
-      MAC2AH:
-        name: MACA2H
-  MACA2LR:
-    _modify:
-      MACA2L:
-        bitWidth: 32
-  MACA3LR:
-    _modify:
-      MBCA3L:
-        name: MACA3L
-
-Ethernet_DMA:
-  DMABMR:
-    _modify:
-      # This field is named incorrectly in the SVD compared to the RM
-      RTPR:
-        name: PM
-
 # Merge the thousands of individal bit fields into a single field for each
 # CAN filter register. This is not only much easier to use but also saves
 # a huge amount of filespace and compilation time etc -- as much as 30% of all
@@ -88,6 +57,7 @@ _include:
  - common_patches/merge_I2C_OAR1_ADDx_fields.yaml
  - common_patches/dma_fcr_wo.yaml
  - common_patches/ethernet_macmiidr.yaml
+ - common_patches/f4_ethernet.yaml
  - ../peripherals/dac/dac_wavegen.yaml
  - ../peripherals/dac/dac_common_2ch.yaml
  - ../peripherals/rcc/rcc_v2.yaml

--- a/devices/stm32f427.yaml
+++ b/devices/stm32f427.yaml
@@ -57,6 +57,7 @@ _include:
  - common_patches/merge_I2C_OAR1_ADDx_fields.yaml
  - common_patches/dma_fcr_wo.yaml
  - common_patches/ethernet_macmiidr.yaml
+ - common_patches/f4_ethernet.yaml
  - ../peripherals/dac/dac_wavegen.yaml
  - ../peripherals/dac/dac_common_2ch.yaml
  - ../peripherals/rcc/rcc_v2.yaml
@@ -69,6 +70,9 @@ _include:
  - ../peripherals/spi/spi_v1.yaml
  - ../peripherals/tim/tim_basic.yaml
  - ../peripherals/tim/tim_gp1.yaml
+ - ../peripherals/eth/eth_mac.yaml
+ - ../peripherals/eth/eth_mmc.yaml
+ - ../peripherals/eth/eth_dma.yaml
  - ../peripherals/usart/uart_common.yaml
  - ../peripherals/usart/uart_uart.yaml
  - ../peripherals/usart/uart_usart.yaml

--- a/devices/stm32f429.yaml
+++ b/devices/stm32f429.yaml
@@ -57,6 +57,7 @@ _include:
  - common_patches/merge_I2C_OAR1_ADDx_fields.yaml
  - common_patches/dma_fcr_wo.yaml
  - common_patches/ethernet_macmiidr.yaml
+ - common_patches/f4_ethernet.yaml
  - ../peripherals/dac/dac_wavegen.yaml
  - ../peripherals/dac/dac_common_2ch.yaml
  - ../peripherals/rcc/rcc_v2.yaml
@@ -69,6 +70,9 @@ _include:
  - ../peripherals/spi/spi_v1.yaml
  - ../peripherals/tim/tim_basic.yaml
  - ../peripherals/tim/tim_gp1.yaml
+ - ../peripherals/eth/eth_mac.yaml
+ - ../peripherals/eth/eth_mmc.yaml
+ - ../peripherals/eth/eth_dma.yaml
  - ../peripherals/usart/uart_common.yaml
  - ../peripherals/usart/uart_uart.yaml
  - ../peripherals/usart/uart_usart.yaml

--- a/devices/stm32f469.yaml
+++ b/devices/stm32f469.yaml
@@ -47,6 +47,7 @@ _include:
  - common_patches/merge_I2C_OAR1_ADDx_fields.yaml
  - common_patches/dma_fcr_wo.yaml
  - common_patches/ethernet_macmiidr.yaml
+ - common_patches/f4_ethernet.yaml
  - ../peripherals/dac/dac_wavegen.yaml
  - ../peripherals/dac/dac_common_2ch.yaml
  - ../peripherals/gpio/gpio_v2.yaml
@@ -57,6 +58,9 @@ _include:
  - ../peripherals/spi/spi_v1.yaml
  - ../peripherals/tim/tim_basic.yaml
  - ../peripherals/tim/tim_gp1.yaml
+ - ../peripherals/eth/eth_mac.yaml
+ - ../peripherals/eth/eth_mmc.yaml
+ - ../peripherals/eth/eth_dma.yaml
  - ../peripherals/usart/uart_common.yaml
  - ../peripherals/usart/uart_uart.yaml
  - ../peripherals/usart/uart_usart.yaml

